gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/zosmf/ZosmfServiceV1.java

```diff
@@ -89,12 +92,12 @@ public void validate(TokenType type, String token) {
 
     @Override
     public void invalidate(TokenType type, String token) {
-        // not supported by z/OSMF
+        // not supported by this version of z/OSMF
     }
 
     @Override
-    public boolean matchesVersion(int version) {
-        return version <= 25;
+    public boolean isSupported(int version) {
```

Comment:
Is it reasonable expectation that the V1 will be always supported?

Reply:
The functionality that this code expects in z/OSMF is planned to be supported forever.